src/sample.json

```diff
@@ -0,0 +1,380 @@
+{
```

Comment:
What is this file for? I don' think your code is using it now.

Reply:
This was a reference file for my own use that I added to `src` by mistake. It might be useful to include in the repo, but it should be in a `reference` folder or just in the root directory.